src/confluent/tools/handlers/schema/list-schemas-handler.ts

```diff
@@ -0,0 +1,62 @@
+import { ClientManager } from "@src/confluent/client-manager.js";
+import { CallToolResult } from "@src/confluent/schema.js";
+import {
+  BaseToolHandler,
+  ToolConfig,
+} from "@src/confluent/tools/base-tools.js";
+import { ToolName } from "@src/confluent/tools/tool-name.js";
+import env from "@src/env.js";
+import { wrapAsPathBasedClient } from "openapi-fetch";
+import { z } from "zod";
+
+const listSchemasArguments = z.object({
+  baseUrl: z
+    .string()
+    .describe("The base URL of the Schema Registry REST API.")
+    .url()
+    .default(env.SCHEMA_REGISTRY_ENDPOINT ?? "")
+    .optional(),
+});
+
+export class ListSchemasHandler extends BaseToolHandler {
+  async handle(
+    clientManager: ClientManager,
+    toolArguments: Record<string, unknown>,
+  ): Promise<CallToolResult> {
+    const { baseUrl } = listSchemasArguments.parse(toolArguments);
+
+    if (baseUrl !== undefined && baseUrl !== "") {
+      clientManager.setConfluentCloudSchemaRegistryEndpoint(baseUrl);
+    }
+
+    const pathBasedClient = wrapAsPathBasedClient(
+      clientManager.getConfluentCloudSchemaRegistryRestClient(),
+    );
+
+    // First get all subjects
+    const { data: response, error: error } = await pathBasedClient[
+      "/schemas"
+    ].GET({
+      query: {
+        latestOnly: "true",
```

Comment:
nit - should this be an input param?

Reply:
yes this input param only pulls the latest for all schemas 
![Screenshot 2025-04-04 at 9 19 13 PM](https://github.com/user-attachments/assets/317dd309-688a-4552-b7ef-ae6e20dcb661)

Reply:
this is a input param to get the latest schemas only

Reply:
got it. changed the implementation to take latestOnly, subjectPrefix and deleted from the intent